tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -127,22 +127,23 @@
                                                 settings={'allow_so_beginning':
                                                           False})
 
-WriteGoodLintBearWithIllusionTest = verify_local_bear(WriteGoodLintBear,
-                                                      valid_files=(
-                                                          good_file,
-                                                          weasel_file,
-                                                          wordy_file,
-                                                          thereIs_file,
-                                                          cliche_file,
-                                                          adverb_file,
-                                                          passive_file,
-                                                          bad_file,
-                                                          so_file,),
-                                                      invalid_files=(
-                                                          illusion_file,),
-                                                      settings={
-                                                        'allow_repeated_words':
-                                                            False})
+WriteGoodLintBearWithIllusionTest = verify_local_bear(
+                                                WriteGoodLintBear,
```

Comment:
(as explained above, you do not need to do this, and shouldnt do this... but ...)

**If** you are going to revise all these lines, you should try to avoid future line length problems by de-denting everything as much as possible.

This line only needs to be indented by four spaces, and the other lines below indented by four or eight spaces.  i.e.

```
WriteGoodLintBearWithIllusionTest = verify_local_bear(
    WriteGoodLintBear,
    valid_files=(
    ....
```